modules/code-editor/src/views/full/FileNavigator.tsx

```diff
@@ -260,13 +258,16 @@ export default inject(({ store }: { store: RootStore }) => ({
 }))(observer(FileNavigator))
 
 type Props = {
+  id: number
```

Comment:
The typing is declared as a number, while you use it as a string everywhere. I suggest converting id to string, and replacing existing numeric ID by a string representing the navigator window (ex: config, modules, etc)